WordPress/gutenberg
# Update border color on color panel items

## What?
The color panel buttons use a slightly different border color than throughout the rest of the sidebar. Just tightening up the interface by using the `$gray-300` value instead of `rgba(0, 0, 0, 0.1)`. 

## How?
Small style tweak

## Testing Instructions
1. Open a Post or Page.
2. Click on a Paragraph Block and open the Settings Sidebar
3. See consistent border color on the Color panel

## Screenshots 
<img width="1276" alt="CleanShot 2022-07-08 at 17 18 15@2x" src="https://user-images.githubusercontent.com/1813435/178071707-923565bb-2cef-4878-a4b5-b29fab4cc223.png">
